pkg/clients/common/namespace.go

```diff
@@ -206,3 +205,25 @@ func (s *SuiteController) namespaceDoesNotExist(namespace string) wait.Condition
 func (s *SuiteController) GetNamespace(namespace string) (*corev1.Namespace, error) {
 	return s.KubeInterface().CoreV1().Namespaces().Get(context.Background(), namespace, metav1.GetOptions{})
 }
+
+// Ensure that the labeles provided in `labels` (including their values) exists on namespace `ns`
+// return true if the namespace was updated
+func (s *SuiteController) ensureLabelsExist(ns *corev1.Namespace, requiredLabels map[string]string) (bool, error) {
+	maps.DeleteFunc(requiredLabels, func(k, v string) bool {
+		val, exists := ns.Labels[k]
+		return exists && ns.Labels[k] == val
```

Comment:
shouldn't this be `return exists && requiredLabels[k] == val` instead?

I think `ns.Labels[k] == val` will be always `true` no matter what